src/storable.rs

```diff
@@ -37,6 +43,12 @@ impl Storable for () {
     }
 }
 
+impl BoundedStorable for () {
```

Comment:
Maybe use a macro for all of these numeric type implementations.
Also is there a reason i8, i16, etc aren't covered?

Reply:
Totally, but one step at a time :)